packages/dds/tree/docs/undo/README.md

```diff
@@ -181,12 +181,13 @@ To that end, we mostly reuse the existing code paths for changesets by always se
 The undo edit is created by inverting the edit that needs to be undone,
 and rebasing that inverse over all the changes that have been applied since.
 
-Sending concrete undo edits alleviates the need establish and maintain distributed consensus on an undo window.
-It also means summaries do not need to include repair data.
+Sending concrete undo edits alleviates the need to establish and maintain distributed consensus on an undo window.It also means summaries do not need to include repair data.
 It does however require sending rebased changes over the wire.
+Rebased changes may contain lineage entries, which we haven't sent over the wire before.
```

Comment:
Even without undo we will have to send rebased changes if the framework asks SharedTree to resubmit an op.